stl/inc/memory

```diff
@@ -1848,6 +1849,7 @@ public:
         return *this;
     }
 
+    template <class _MyDx = _Dx, enable_if_t<is_move_constructible_v<_MyDx>, int> = 0>

```

Comment:
This should be `is_­move_­assignable_­v` according to https://eel.is/c++draft/unique.ptr.single.asgn#1 .